src/status_im2/contexts/chat/lightbox/text_sheet/utils.cljs

```diff
@@ -8,23 +8,32 @@
 
 (defn sheet-gesture
   [{:keys [derived-value saved-top overlay-opacity gradient-opacity]}
-   expanded-height max-height overlay-z-index expanded? dragging?]
+   expanded-height max-height full-height overlay-z-index expanded? dragging? expanding-message?]
   (-> (gesture/gesture-pan)
+      (gesture/enabled expanding-message?)
       (gesture/on-start (fn []
                           (reset! overlay-z-index 1)
                           (reset! dragging? true)
-                          (reanimated/animate gradient-opacity 0)))
+                          (when (not expanded?)
+                            (reanimated/animate gradient-opacity 0))))
       (gesture/on-update
        (fn [e]
-         (let [new-value     (+ (reanimated/get-shared-value saved-top) (oops/oget e "translationY"))
-               bounded-value (max (min (- new-value) expanded-height) constants/text-min-height)
-               progress      (/ (- new-value) max-height)]
-           (reanimated/set-shared-value overlay-opacity progress)
-           (reanimated/set-shared-value derived-value bounded-value))))
+         (let [event-value     (oops/oget e "translationY")
```

Comment:
We can use keywords too as the args to `oget`. The only practical advantage I can think of is that keywords are allocated only once, so `:translationY` would be allocated once and reused.